flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/config/MySqlSourceConfigFactory.java

```diff
@@ -72,6 +73,7 @@ public class MySqlSourceConfigFactory implements Serializable {
     private Duration heartbeatInterval = HEARTBEAT_INTERVAL.defaultValue();
     private Properties dbzProperties;
     private String chunkKeyColumn;
+    private List<String> specificPks = new ArrayList<>();
```

Comment:
Could current design cover the case which user captures two NoPKTable with different speciffic primary keys(talbe 1: cola,colb, table 2：col1, col2)?